settings.gradle.kts

```diff
@@ -1 +1,5 @@
 rootProject.name = "hmpps-template-kotlin"
+
+plugins {
+  id("org.gradle.toolchains.foojay-resolver-convention") version("0.4.0")
```

Comment:
I initially tried to put this in `build.gradle.kts` which didn't work, then found https://github.com/gradle/foojay-toolchains/issues/15#issuecomment-1377402790 hence it being here